atuin/src/command/client/history.rs

```diff
@@ -34,6 +34,8 @@ pub enum Cmd {
         id: String,
         #[arg(long, short)]
         exit: i64,
+        #[arg(long, short, default_value = "-1")]
+        duration: i64,
```

Comment:
I'd rather we make this an option, it avoids the -1 special casing and ensures this is properly represented in the type system. We can then make it a u64

```suggestion
        #[arg(long, short)]
        duration: Option<u64>,
```